src/Transition/CreateNewIDFUsingRulesV9_1_0.f90

```diff
@@ -423,6 +423,21 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
               ! If your original object starts with Z, insert the rules here
 
+              CASE('ZoneHVAC:EquipmentList')
+                  CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                  nodiff=.false.
+                  OutArgs(1) = InArgs(1)
+                  OutArgs(2) = InArgs(2)
+                  DO I = 1, (CurArgs - 2)/4
```

Comment:
Maybe a naive question here, but can `(CurArgs-2)` be a non-multiple of 4?  And if so will this give some unexpected behavior?

Reply:
It _should_ always be a multiple of 4 because in the original idd there are two arguments followed by an extensible group with 4 items where each item is required.

Reply:
OK, if the 4 was required by IDD markup in the prior files, the transition program will be aware of that and warn the user about it.